### PR TITLE
Add RocketCake.app latest

### PR DESCRIPTION
**Pull request by proinsias**
RocketCake is a free web editor for creating responsive websites.

### CODE REVIEW

File: Casks/rocketcake.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'rocketcake' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://www.ambiera.at/downloads/RocketCake-1.0.dmg'
```
Comment (adidalal) [inline Casks/rocketcake.rb]: Use #{version} and update values accordingly.

**Comment by adidalal** [comment]
Should be 1.0.4, and use can use version.major_minor.

zap: `~/Library/Application Support/com.apple.sharedfilelist/com.apple.LSSharedFileList.ApplicationRecentDocuments/com.ambiera.rocketcake.sfl`

**Comment by proinsias** [comment]
Done!

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 41f1b5b7546b9d047014bac3dcbaae042a6f4b1d to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**